### 1. It is claimed that the amount of cholesterol in the blood exceeds 180 in people who do sports. Cholesterol levels of randomly selected people who do sports are given. Test this claim at the 5% significance level.

In [5]:
library(readxl)
library(BSDA)

In [2]:
df <- read_excel("/data/notebook_files/Cholesterol_Data.xlsx")

In [3]:
head(df)

A tibble: 6 × 1 
 
 cholesterol 
 <dbl> 
 
 
 201 
 188 
 167 
 155 
 234 
 170

##### Hypothesis testing for the population mean of cholesterol amounts;
The z-test (with sample variance) is used because the mass variance (𝜎2) is unknown and n>30.

                                                        H0: µ=180
                                                        
                                                        H1: µ>180

In [6]:
z.test(df, mu=180, alternative="greater", sigma.x=sqrt(var(df)))


	One-sample z-Test

data:  df
z = 1.6805, p-value = 0.04643
alternative hypothesis: true mean is greater than 180
95 percent confidence interval:
 180.1525       NA
sample estimates:
mean of x 
 187.1905 


### Conclusion:
#### i) Z Test Statistic:
H0 is rejected as Z=1.6805 > Z0.05 =1.645
#### ii) p value
H0 is rejected as p=0.04643 < α=0.05
#### iii) Confidence interval
Confidence Interval: P(180.1525<μ )=0.95

H0 is rejected because the confidence interval does not include value 180.
#### Interpretation:
Since H0 is rejected at the 5% significance level, it can be concluded that the amount of cholesterol in blood It is not 180 (there is insufficient evidence that it is 180).